src/devtools/client/debugger/src/utils/sourceVisualizations.ts

```diff
@@ -131,6 +131,25 @@ export function getSourcemapVisualizerURLSuspense(
   return href;
 }
 
+export function getSourcemapsViewerURLSuspense(
```

Comment:
Why is this called "Suspense"? It's not suspending.
```suggestion
export function getSourcemapsViewerURL(
```